hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/EntityDaoModule.java

```diff
@@ -65,6 +65,12 @@ protected void configure() {
             new TypeLiteral<
                 Converter<Collection<AttributeAssociation<FilterArgument>>, Filter>>() {}));
 
+    requireBinding(
+        Key.get(
+            new TypeLiteral<
+                Converter<
+                    Collection<Collection<AttributeAssociation<FilterArgument>>>, Filter>>() {}));
```

Comment:
Please address https://github.com/hypertrace/hypertrace-core-graphql/pull/151#discussion_r1406187848 before proceeding with this PR